doc/dev/release-checklists.rst

```diff
@@ -145,7 +138,9 @@ After dev freeze
 ================
 
 - [ ] create branch for new release
-- [ ] remove release notes for release branch: see also 33d63c32cbf81ed81ce2185063011b98846a0d44
+- [ ] remove ``doc/releases/*.rst``. This should leave behind ``doc/releases/releases.yml`` which is used for doc building purposes.
+- [ ] cherry-pick 8cf9ad62949516666ad0f2c0bb7726ef68e4d666 ("doc: add releases links to toc"). There will be trivial conflicts.
+- [ ] add redirect for new major release at `RTD <https://readthedocs.org/dashboard/ceph/redirects/>`_.
```

Comment:
Just want to double check that it's indeed the case and that there is no way to do set it up just once with some sort of wild card?

Reply:
I tried wildcards in the past and it didn't work. The configuration options are somewhat clunky and primitive.